src/code/__osMalloc.c

```diff
@@ -746,7 +746,7 @@ void ArenaImpl_FaultClient(Arena* arena) {
     ArenaNode* next;
 
     FaultDrawer_Printf("ARENA INFO (0x%08x)\n", arena);
-    if (!__osMallocIsInitalized(arena)) {
+    if (!__osMallocIsInitialized(arena)) {
         FaultDrawer_Printf("Arena is uninitalized\n", arena);
```

Comment:
Probably from here.

Reply:
Yeah I noticed that but surely the 30 typos of "initialized" in the repo weren't on purpose to honor this typo right? :fearful:

Reply:
I'm guessing it was one of the first files and no one else knew how to spell it lol.